.github/workflows/publish-flutter.yml

```diff
@@ -78,24 +85,30 @@ jobs:
 
       - uses: actions/download-artifact@v4
         with:
-          name: bindings-flutter
-          path: bindings/
+          name: bindings-swift
+          path: dist/ios/Sources/BreezSDKLiquid/
+
+      - name: Move Source header files
+        working-directory: dist/ios/Sources/BreezSDKLiquid/
+        run: |
+          mv breez_sdk_liquidFFI.h ../../Classes
```

Comment:
I think this is covered above now, but we should still remove the modulemap

Reply:
Done. 
[cleanup header file artifacts](https://github.com/breez/breez-sdk-liquid/pull/510/commits/dcd92544562d6d9198ba8f9c445523e13c5b8a7b)